core/src/main/java/org/apache/brooklyn/core/entity/BrooklynConfigKeys.java

```diff
@@ -88,40 +93,48 @@
      * <p>
      * If this key is set on a {@link Location} then all entities in that location will be treated in this way, again as with {@link #ENTITY_STARTED}.
      *
-     * @see #ENTITY_STARTED
+     * @see #SKIP_ENTITY_START
      */
-    public static final ConfigKey<Boolean> SKIP_ENTITY_START_IF_RUNNING = newBooleanConfigKey("entity.running", "Skip the startup process entirely, if service already running");
+    public static final ConfigKey<Boolean> SKIP_ENTITY_START_IF_RUNNING = ConfigKeys.builder(Boolean.class)
+            .name("skip.start.ifRunning") 
+            .deprecatedNames("entity.running") 
+            .description("Whether to skip the startup process entirely, but only if it already running")
```

Comment:
`Whether to skip the startup process if the entity is detected as already running` ?